backend/src/main/java/gov/cdc/usds/simplereport/api/queue/QueueMutationResolver.java

```diff
@@ -34,22 +36,27 @@ public QueueMutationResolver(TestOrderService tos, PersonService ps) {
       _ps = ps;
     }
 
-    public void addTestResult(String deviceID, String result, String patientID, String dateTested) {
```

Comment:
If we haven't tried it, let's try to make this `Date dateTested` (with corresponding schema change of type to `DateTime`) and let somebody else do the parsing for you. 😺

Reply:
Resolution: wait for an API clean-up job.